frontends/web/src/components/confirm/Confirm.jsx

```diff
@@ -25,6 +25,7 @@ export default class Confirm extends Component {
     }
 
     componentDidMount() {
+        // @ts-ignore (override changes the method signature)
```

Comment:
How about making `this.confirmOverride` conform to the signature of `window.confirm` instead?
Maybe in a follow up change or create a bug?

Reply:
Yes, I both mentioned it in our internal communication channel and already created an internal issue yesterday. I just wanted to keep the commit as focussed as possible/reasonable (and let other clean up their own mess).

Reply:
Yeah, I saw the comm but missed the bug. Just wanted to make sure. Thanks!